doc/classes/@GlobalScope.xml

```diff
@@ -877,6 +877,7 @@
 				[/codeblocks]
 				[b]Note:[/b] Consider using [method push_error] and [method push_warning] to print error and warning messages instead of [method print] or [method print_rich]. This distinguishes them from print messages used for debugging purposes, while also displaying a stack trace when an error or warning is printed.
 				[b]Note:[/b] On Windows, only Windows 10 and later correctly displays ANSI escape codes in standard output.
+				[b]Note:[/b] Output displayed in the editor supports clickable [code][url=address][/code] tags. The [code][url][/code] tag's [code]address[/code] value is handled by [method OS.shell_open] when clicked.
```

Comment:
Otherwise this will likely raise warnings from `make_rst.py`.
```suggestion
				[b]Note:[/b] Output displayed in the editor supports clickable [code skip-lint][url=address][/code] tags. The [code skip-lint][url][/code] tag's [code]address[/code] value is handled by [method OS.shell_open] when clicked.
```

Reply:
This actually seems to work fine right now because there's no closing `[/url]` tag within `[code]`, but I didn't know about `skip-lint`.

I'll add it either way :slightly_smiling_face: